portfolio/src/main/webapp/script.js

```diff
@@ -155,3 +155,17 @@ async function checkLogin(){
     document.getElementById("login-link").href = details.loginUrl;
   }
 }
+
+/** Creates a map and adds it to the page. */
+function initMap() {
+  const nightMap = new google.maps.StyledMapType(nightVersion, {name: 'Night'});
+  const map = new google.maps.Map( document.getElementById("map"), {
```

Comment:
Nit: Both of these variables have suffix `map` that kinda indicates that they represent the same type of object - which is not true I guess. Could you please change the name of the first variable a bit? I guess it represents some _type_ or _style_ and hence you can probably rename it to `nightMapType` or `nightMapStyle`? 

This is a rule of thumb that the last part of a variable name should say what the variable is basically.